lib/tdf3/src/tdf.ts

```diff
@@ -401,7 +401,7 @@ export async function writeStream(cfg: EncryptConfiguration): Promise<DecoratedR
     cfg.encryptionInformation,
     cfg.policy,
     cfg.mimeType,
-    cfg.tdfSpecVersion ?? '4.3.0'
+    cfg.tdfSpecVersion
```

Comment:
The generateManifest would use 
```
schemaVersion: targetSpecVersion || tdfSpecVersion
```

And `tdfSpecVersion` is `4.3.0`. 

Removing the `4.3.0` is better,  does it fix a bug or its cleanup?

Reply:
Yeah, sorry there are a few small cleanups mixed in with the fix. The fix is just the change in lib/tdf3/src/client/index.ts

Reply:
Awesome! This is fantastic refactor :)